content/post/the-lab-book.md

```diff
@@ -0,0 +1,87 @@
+---
+author: Michael Kaminsky
+title: "The Data Science Lab Book"
+cover: "/img/agile-analytics-p1.jpg"
+publishDate: "2019-11-30"
+tags: 
+ - data science 
+ - process
+ - tools
+---
+
+For the last year or so I’ve been working on building a software application to help marketers allocate their marketing spend. This software is statistics and data-science powered and my partner and I have spent more hours than I’d like to admit struggling to squash bugs, achieve model convergence, and generally answer the question “why on earth could *_that_* be happening?”
+
+In this post I’ll discuss the history of the lab book and how it’s used generally before discussing how to use it for data science and software engineering projects and providing an example lab book template.
+
+## What is the lab book?
+
+The lab notebook has been around, mostly unchanged, for hundreds of years. The lab notebook is, most simply, a record of what happens in the lab as a researcher is running experiments. It captures the researcher’s thoughts and ideas, their hypotheses, and their learnings.
+
+If you recall the famous (and perhaps apocryphal) Thomas Edison quote “I have not failed. I've just found 10,000 ways that won't work” you surely must imagine that he kept track of those 10,000 attempts somewhere. The place for tracking those attempts, learning what won’t work, and (hopefully) coming up with ideas for what will, takes place in the lab book.
+
+At the most general level, a lab book should have the following qualities:
+
+* It should record the context of an experiment: the hypothesis being tested and the results observed
+* It should be readable by others: other members of the team should be able to read the entries and be able to replicate the experiment and obtain the same results
+
+Any conclusions that are drawn from a given experiment should reference supporting evidence in the lab book.
+
+## The lab book for data scientists
```

Comment:
May be interesting to talk a bit more about how a lab notebook is different from a traditional lab notebook.